<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transformations3DWidget</class>
 <widget class="QWidget" name="Transformations3DWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1535</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Transformations3DCanvas" name="transformations3DCanvas" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1100</width>
     <height>800</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="translationLabel">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Translacja</string>
   </property>
  </widget>
  <widget class="QSlider" name="translationXSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>70</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="translationYSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>110</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="translationZSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>150</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="translationXLabel">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>60</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="translationYLabel">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="translationZLabel">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>140</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="scalingLabel">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Skalowanie</string>
   </property>
  </widget>
  <widget class="QLabel" name="scalingZLabel">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>320</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QSlider" name="scalingYSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>290</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="scalingXLabel">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>240</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QSlider" name="scalingXSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>250</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="scalingYLabel">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>280</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QSlider" name="scalingZSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>330</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="rotationLabel">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>380</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Rotacja</string>
   </property>
  </widget>
  <widget class="QLabel" name="rotationZLabel">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>500</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QSlider" name="rotationYSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>470</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="rotationXLabel">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>420</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QSlider" name="rotationXSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>430</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="rotationYLabel">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>460</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QSlider" name="rotationZSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>510</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="perspectiveLabel">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>550</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Perspektywa</string>
   </property>
  </widget>
  <widget class="QLabel" name="perspectiveFLabel">
   <property name="geometry">
    <rect>
     <x>1460</x>
     <y>590</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QSlider" name="perspectiveSlider">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>600</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Transformations3DCanvas</class>
   <extends>QWidget</extends>
   <header>transformations3dcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
